src/sections/Landscape-grid/smi.js

```diff
@@ -40,19 +40,10 @@ function SMI_Compatibility() {
   return (
     <Tabs style={{ overflow: "auto", whiteSpace: "nowrap"}} className="landscape-table">
       <TabList>
-        <Tab>V0.5.0</Tab>
-        <Tab>V0.6.0</Tab>
-        <Tab>V0.4.0</Tab>
+        <Tab>V0.1.0</Tab>
```

Comment:
Is this SMI's version number or the version number of our test suite? @dhruv0000 @kumarabd

Reply:
Test suite

Reply:
Thank you, sir.

Reply:
Proposal: let's have the test suite version numbers follow/be that of Learn Layer5's version numbers - https://github.com/layer5io/learn-layer5/releases. What do you think?

Reply:
Would have made sense if it was an independent repo. How will changes not related to smi-tests affect the table?

Reply:
You're right - those changes may not affect the table. Other changes to the test criteria may or may not affect the table either. Changes to either the test criteria or the test tool (and it's behavior) warrant a bump in version number as changes to the tool or changes to the criteria can both affect test results.